prow/config/jobs_test.go

```diff
@@ -654,7 +675,7 @@ func TestMergePreset(t *testing.T) {
 		},
 	}
 	for _, tc := range tcs {
```

Comment:
We need to make sure Steve stays warm:

```golang
for _, tc := range tcs {
  t.Run(tc.name, func(t *testing.T) {
  }
}
```

Then you can remove all the `for test \"%s\"` stuff as this happens automatically

Reply:
:+1: done